Day1/puzzle_two.py

```diff
@@ -1,18 +1,9 @@
+import itertools
 
-expense_list = []
 with open("day_one.txt", 'r') as puzzle_input:
-    for line in puzzle_input:
-        expense_list.append(int(line))
+    expense_list = [int(line) for line in puzzle_input]
 
-first_pass = set(expense_list)
-second_pass = dict()
-
-for cost in expense_list:
-    for price in first_pass:
-        second_pass[cost+price] = [cost, price]
-
-for cost in expense_list:
-    for sum in second_pass:
-        if cost + sum == 2020:
-            print(f"answers are {cost}, {second_pass[sum][0]} and {second_pass[sum][1]}")
-            print(f"multiplying gives {cost*second_pass[sum][0]*second_pass[sum][1]}")
+for a, b, c in itertools.combinations(expense_list, 3):
```

Comment:
I've cheekily replaced the entire solution with "use the standard library, Luke".

But you were wasting the power of the dictionary: if you did a dictionary lookup, rather than `for sum in second_pass`, you would go from cubic to quadratic (mumble mumble dictionary lookup isn't *actually* constant).